tests/class.py

```diff
@@ -265,14 +436,62 @@ def test_BytesHeaderParser_of_email_parser(self):
     self.assertOnlyIn((3, 3), detect("from email.parser import BytesHeaderParser"))
 
   def test_EmailPolicy_of_email_policy(self):
-    self.assertOnlyIn((3, 6), detect("from email.policy import EmailPolicy"))
+    self.assertOnlyIn((3, 3), detect("from email.policy import EmailPolicy"))
```

Comment:
Actually I have no choice but to do so. Otherwise `"email.policy.EmailPolicy.content_manager": (None, (3, 4))` will contradict with `"email.policy.EmailPolicy": (None, (3, 6))`.

Reply:
Makes sense.